app/models/sale.rb

```diff
@@ -359,4 +360,8 @@ def has_document?
   def default_values
     self.payment_term = DEFAULT_PAYMENT_TERM
   end
+
+  def parent_name
+    '#'+id.to_s
```

Comment:
Shouldn't this be the `invoice_number`?
We never show the id from the database to the user anywhere else...